src/apify_client/clients/base/base_client.py

```diff
@@ -47,3 +47,13 @@ def _params(self, **kwargs: Any) -> Dict:
         merged_params = self.params.copy()
         merged_params.update(kwargs)
         return merged_params
+
+    def _sub_resource_init_options(self, **kwargs: Any) -> Dict:
+        options = {
+            "base_url": self.url,
+            "http_client": self.http_client,
+            "params": self.params,
+        }
+
+        options.update(kwargs)
+        return options
```

Comment:
Wouldn't the following be easier to read?
```
return {
     ...options,
     ...kwargs
}
```

Reply:
Just as the other similar note - slightly less efficient, though it doesn't matter here. 
I will change this one, since your way is prettier in this case :)